NBReorderTableView/NBReorderTableView.m

```diff
@@ -135,7 +135,7 @@ - (void)startMovingCellAtLocation:(CGPoint)location
     NSIndexPath *indexPath = [self indexPathForRowAtPoint:location];
     UITableViewCell *cell = [self cellForRowAtIndexPath:indexPath];
     
-    if (cell == nil) {
+    if (cell == nil || indexPath == nil) {
```

Comment:
This isn't actually protecting anything because if `indexPath` is nil, then `cell` will also be nil.